ThreeBotPackages/threefold/communityapp/chatflows/community_join.py

```diff
@@ -20,33 +19,137 @@ def chat(bot):
         email=user_email, name=user_name, referral=bot.kwargs.get("referral")
     )
 
-    gedis_client.actors.community_manager.set_current_user(user=user_name)
+    if not invited:
```

Comment:
may instead of 
```
if not invited:
...
if invited:
...
```
make it:
```
if invited:
...:
else:
....
```

Reply:
i need not inivited first in chatflow , to try another solution and if still not invited send mail to the user